sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/README.md

```diff
@@ -11,9 +11,63 @@ description: Samples for the Azure.Messaging.EventHubs.Processor client library
 
 # Azure.Messaging.EventHubs.Processor Samples
 
-The samples are accompanied by a console application which you can use to execute and debug them interactively.  The simplest way to begin is to launch the project for debugging in Visual Studio or your preferred IDE and provide the Event Hubs connection information in response to the prompts.
+The  Azure Event Hubs Processor samples are intended to serve as an example and introduction to common scenarios in which the Event Hubs Processor client library is used, and to help demonstrate library features.  The samples are accompanied by a [console application](./Program.cs) which you can use to execute and debug them interactively.  The simplest way to begin is to launch the project for debugging in Visual Studio or your preferred IDE and provide the Event Hubs connection information in response to the prompts.
 
 Each of the samples is self-contained and focused on illustrating one specific scenario.  Each is numbered, with the lower numbers concentrating on basic scenarios and building to more complex scenarios as they increase; though each sample is independent, it will assume an understanding of the content discussed in earlier samples.
 
-- [Hello world](./Sample01_HelloWorld.cs)
-  An introduction to processing events using the Event Processor, illustrating how to configure the processor and connect to the Event Hubs service.
+## Getting started
+
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Event Hubs, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+- **Event Hubs namespace with an Event Hub:** To interact with Azure Event Hubs, you'll also need to have a namespace and Event Hub available.  If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [creating an Event Hub using the Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create).  There, you can also find detailed instructions for using the Azure CLI, Azure PowerShell, or Azure Resource Manager (ARM) templates to create an Event Hub.
+
+- **Azure Storage account with blob storage:** To persist checkpoints as blobs in Azure Storage, you'll need to have an Azure Storage account with blobs available.  If you are not familiar with Azure Storage accounts, you may wish to follow the step-by-step guide for [creating a storage account using the Azure portal](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&tabs=azure-portal).  There, you can also find detailed instructions for using the Azure CLI, Azure PowerShell, or Azure Resource Manager (ARM) templates to create storage accounts.
```

Comment:
Not sure either, it was a direct copy/paste for the link.   It *seems* to work fine without it, but since we don't own the documentation site and can't predict what it will do... I'm going to just leave this be.   I'm probably being overly paranoid, but ....